README.md

```diff
@@ -124,8 +124,8 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 * [Watchtower](https://github.com/v2tec/watchtower) - Automatically update running Docker containers by [@CenturyLinkLabs][CenturyLinkLabs]
 
 ### CI Services
+* [CodeFresh](https://codefresh.io) - The Fastest CI/CD Platform for Docker Users.
```

Comment:
does not respect the lexical order. Everything is ordered as the letter of the alphabet in English (as everything is in English here)